yarn.lock

```diff
@@ -1451,6 +1451,11 @@
     "@types/minimatch" "*"
     "@types/node" "*"
 
+"@types/history@*":
+  version "4.7.2"
+  resolved "https://registry.npmjs.org/@types/history/-/history-4.7.2.tgz#0e670ea254d559241b6eeb3894f8754991e73220"
```

Comment:
Use the verdaccio registry instead. 

https://registry.verdaccio.org/

So, if npm fails (404) .. we cache all and we can keep working. 😄